src/helpers/modifyDeck.js

```diff
@@ -0,0 +1,50 @@
+export default (deck, modifier) => {
+  switch (modifier) {
+    case '1':
+      return deck.map(card =>
+        card.type === 'structure'
+          ? { ...card, mana: 2, costReduced: card.mana > 2 ? true : false }
+          : card
+      )
```

Comment:
I would recommend writing small functions to make this a little easier to read. Something like:

```js
const reduceStructureManaCost = card => {
  if (card.type !== 'structure') return card
  return { ...card, mana: 2, costReduced: true }
}

export default (deck, modifier) => {
  switch (modifier) {
    case '1':
      return deck.map(reduceStructureManaCost)
```